README.md

```diff
@@ -31,6 +31,20 @@ Alternatively, you can run **rebase-helper** in a container:
 
 See [docker reference](https://rebase-helper.readthedocs.io/en/latest/user_guide/rebasing_in_container.html) for more information.
 
+## What do I get from **rebase-helper**?
```

Comment:
All other headings refer to `rebase-helper` as `it`, I don't think this one should be an exception. Alternatively you want to replace `it` with `rebase-helper` in all the headings.